packages/sdk/utils/messaging.ts

```diff
@@ -1,14 +1,14 @@
 import type { RequestHashmap, UnknownObject } from '@walless/core';
 
 let initialized = false;
-let queueInterval;
+let queueInterval: NodeJS.Timer | number | undefined;
 const requestHashmap: RequestHashmap = {};
 
 const runInterval = () => {
 	const queueSize = Object.keys(requestHashmap).length;
 
 	if (queueSize === 0) {
-		clearInterval(queueInterval);
+		clearInterval(queueInterval as number);
```

Comment:
don't cast `as number` it won't solve the root cause, use this instead:
```typescript
if (!!queueInterval && queueSize === 0) {
   // clear it
}
```

Reply:
cast type `as never`